### PR TITLE
German language

### PR DESCRIPTION
**Pull request by NoNamePro0**
This will add the German language to the program.

I'm up that update the file if new Features come with new needed Translations.

### CODE REVIEW

File: src/scripts/i18n/de.json [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    "loadMore": "Lade mehr",
+    "dangerButton": "Bestätige {action}?",
+    "confirmMarkAll": "Möchtest Du wirklich alle Artikel als gelesen markieren?",
+    "confirm": "Bestätigen",
+    "cancel": "Abbrechen",
+    "time": {
+        "now": "gerade eben",
+        "m": " Minuten",
+        "h": "h",
+        "d": " Tage",
```
Comment (yang991178) [inline src/scripts/i18n/de.json:25]: Is there any specific reason for the acronym of minute and day to be written in full?

**Comment by yang991178** [comment]
Thanks for the translation! Other than the above issue, I am ready to include this in the next build.

File: src/scripts/i18n/de.json (hunk 1 above)
Comment (NoNamePro0) [reply src/scripts/i18n/de.json:25]: German has not really a abbreviation for Minutes and Days.

Comment (yang991178) [reply src/scripts/i18n/de.json:25]: I'll leave this as-is for now. Hope the length of the time won't interfere with the UI too much.

File: src/scripts/i18n/de.json [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+        "loadFull": "Lade kompletten Inhalt",
+        "notify": "Benachrechtige wenn im Hintergrund erhalten",
+        "dontNotify": "Nicht benachrechtigen"
+    },
+    "context": {
+        "share": "Teilen",
+        "read": "Lesen",
+        "copyTitle": "Kopiere Title",
+        "copyURL": "Kopiere URL Link",
+        "copy": "Kompieren",
```
Comment (actionschnitzel) [inline src/scripts/i18n/de.json:76]: Please remove this TypeO : "copy": "Kompieren", -> "copy": "Kopieren",

**Pull request merged**